docker/Makefile.image_names

```diff
@@ -0,0 +1,58 @@
+
+# update this if you need everything to rebuild for backwards-incompatible changes
+VERSION = v0.3.0
+GCR_URL ?= us.gcr.io/vcm-ml
+CUDA ?= n
+TEST_ARGS ?=-v -s -rsx
+FORTRAN_NAME = fv3core-build-fortran
+WRAPPER_NAME = fv3core-build-wrapper
```

Comment:
I want to suggest removing all references to fv3gfs-wrapper from this initial version. We only plan to run the wrapper alongside the dycore only, and this is already supported within the fv3core repository. There isn't a use case where we'd need to run fv3gfs-wrapper, fv3core, and fv3gfs-physics together.

If we decide later to better support running the wrapper with our gt4py code we could change this, but as it is this would be a lot of infrastructure dedicated to a configuration we've never had in CI.